Strategies/IITB_KPIs.md

```diff
@@ -52,25 +94,27 @@ Rate of delivery is the measurement of IITB's ability to deliver projects and so
+1\. the rate at which intake capacity increases. The PMO's [Active Project List](http://dialogue/grp/ippst/Approved%20Projects%20Lists%20%28APL%29/Forms/AllItems.aspx?RootFolder=%2Fgrp%2Fippst%2FApproved%20Projects%20Lists%20%28APL%29%2FFY%202019%2D2020&FolderCTID=0x012000D8E2E348AB0A034886495AE4BED12E07&View=%7b2C779E42-EB28-444B-AACD-39CC61DDBCE2%7d), for Major/Minor/Small ESDC projects, as well as the PMO's [IITB Lite projects](http://dialogue/grp/ITPM-GPTI/Lists/IITBPMP%20Small%20Projects/AllItems.aspx) will be used as reference points to monitor intake increase. The project's complexity will be considered in the measurement.
+
+For each 4 complexity and risk levels ([PCRA](https://www.canada.ca/en/treasury-board-secretariat/services/information-technology-project-management/project-management/project-complexity-risk-assessment-tool.html)):
 
-`rate of project delivery = (Projects Year_n - Projects Year_n-1) / Projects Year_n-1`
+`rate of project delivery = (projects year_n - projects year_n-1) / projects year_n-1`
 
-2. the rate at which product releases increases. The list of products and their release schedule is found **&lt;insert location here&gt;**
+2\. the rate at which product releases increases. The list of products and their release schedule is found **&lt;insert location here&gt;**
 
-`rate of product delivery = (Product releases Year_n - Product releases Year_n-1) / Product releases Year_n-1`
+`rate of product delivery = (product releases year_n - product releases year_n-1) / product releases year_n-1`
 
-## 6. **Reduction in Cyber-incidents**
+## 6. **Reduction in cyber incidents**
 
 This indicator will monitor the number of cyber-incidents across ESDC and measure the rate at which they decrease.
 
-`rate = (Incidents Year_n - Incidents Year_n-1) / Incidents Year_n-1`
+`rate of cyber incidents = (incidents year_n - incidents year_n-1) / incidents year_n-1`
 
 ## 7. **Increase cyber resilience**
 
 Cyber resilience is the ability to quickly mitigate and remediate a breach, whether it be from human error or insecure software resulting in operational disruption, data theft or public trust degradation. Cyber resilience will be measured by a decrease rate of intrusion, which is a complex measurement using security frameworks like the [Mitre Att&amp;ck Matrix](https://attack.mitre.org/).
 
-`rate = TBD`
+`rate of cyber resilience = TBD`
```

Comment:
Some thoughts on - 
Cyber resilience may be measured on “not losing operation capability” or “not losing protected information” in an event of cyber attack. 
Situational awareness about our systems could be part of cyber resilience.
From an article – “The challenge of Cyber Resilience can be expressed as a simple question: When security controls fail, can you detect unusual or irregular behaviour with sufficient context to mitigate the risk to the organisation?”
Cyber resilience can be how soon the operational capability was restored?

Reply:
Great thoughts! Resilience could be measured during outage testing (e.g. if we adopt Chaos Engineering practice). I'll merge for now but feel free to submit a PR with proposed formula(s) and we'll discuss it :)

Reply:
Chaos Engineering. That may well be a practice that EO (enterprise operations) should be doing and funded for. Maybe even as part of IT Security program.